### DISCUSSION TITLE
Add `current_status` to Humidifier entity

### DISCUSSION THREAD

**Discussion by Shulyaka** [home-assistant/architecture#930] [Entity Models]
In spite of #856, I'd like to discuss another attribute that might be useful to include into `humidifier` entity. It's similar to `hvac_action` for `climate` entity that shows the current status: cooling, heating, or off. I would like to bring this feature to humidifiers, i.e. make available the current status. I imagine it as a `current_status` attribute with two possible values: `idle` or `active`.

At first I thought it could just be a separate binary or text sensor, but on the second thought I realized that:

1. Users sometimes expect `humidifier` to be similar to `climate` with respect to the features they provide
2. I think this is applicable to many humidifiers/dehumidifiers, they constantly switch on and off in order to reach the target humidity.
3. This is directly related to the humidifying function of the device
4. A separate sensor might be too out of "wack" to hook up into frontend humidifier card, though not impossible

What is your view on this subject?

**Comment by frenck** [comment]
Closing as a duplicate of https://github.com/home-assistant/architecture/discussions/731

**Reply by Shulyaka** [reply]
Oops, forgot about that!
